contrib/coreos/user-data.example

```diff
@@ -19,7 +9,48 @@ coreos:
     etcd_request_timeout: 3.0
   units:
   - name: etcd.service
+    mask: true
+  - name: etcd2.service
     command: start
+    content: |
+      [Unit]
+      Description=etcd2 container
+      Requires=early-docker.service
+      After=early-docker.service
+      Before=early-docker.target
+
+      [Service]
+      User=etcd
+      PermissionsStartOnly=true
+      Restart=always
+      RestartSec=10s
+      LimitNOFILE=40000
+      EnvironmentFile=/etc/environment
+      Environment="ETCD_VERSION=v2.0.13"
+      Environment="ETCD_ELECTION_TIMEOUT=2000"
+      Environment="ETCD_HEARTBEAT_INTERVAL=400"
+      Environment="ETCD_DATA_DIR=/var/lib/etcd2"
+      Environment="ETCD_NAME=%m"
+      Environment="DOCKER_HOST=unix:///var/run/early-docker.sock"
+      ExecStartPre=-/usr/bin/docker kill etcd2
+      ExecStartPre=-/usr/bin/docker rm etcd2
```

Comment:
> Is it necessary to rm and re-pull the container every time the service is started (i.e. on each reboot)?

No, it can follow the same pattern that the rest of the deis services. Sorry for the change in the format.
```
ExecStartPre=/bin/sh -c "ETCD_IMAGE=quay.io/coreos/etcd:$ETCD_VERSION && docker history $ETCD_IMAGE >/dev/null 2>&1 || docker pull $ETCD_IMAGE"
ExecStartPre=/bin/sh -c "docker inspect etcd2 >/dev/null 2>&1 && docker rm -f etcd2 || true"
```